db/migrate/20240111063432_add_micropost_to_react.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# add micropost id
+class AddMicropostToReact < ActiveRecord::Migration[7.1]
+  def change
+    add_column :reacts, :micropost_id, :string
```

Comment:
In a PR, you can use `rails db:rollback` if you miss some fields in migrate file.
Please don't write so many files migrate for a table.